docs/reference/changelog.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 *********
 
+4.2.4 (2024-Sep-19)
```

Comment:
In this case don't you want to target the PR to `hotfix/4.2`?

Reply:
Yes, thanks!